README.md

```diff
@@ -58,20 +58,24 @@ For the latest information about Source-Build support for new .NET versions, ple
 
 ### Code flow
 
-For the time being, the source code only flows one way - from the development repos into the VMR.
-More details on this process:
+The VMR's code flow now operates in two directions. Indivdual repositories flow source changes into the VMR upon promotion of their local official builds (forward flow). The VMR changes are checked in, an official build happens, and then source changes + packages flows backward into the constituent repositories (back flow). For more details on code flow and code flow pull requests, please see this information on [Code Flow PRs](src/arcade/Documentation/UnifiedBuild/Codeflow-PRs.md).
```

Comment:
>The VMR's code flow now operates in two directions. Indivdual repositories flow source changes into the VMR upon promotion of their local official builds (forward flow). The VMR changes are checked in, an official build happens, and then source changes + packages flows backward into the constituent repositories (back flow). For more details on code flow and code flow pull requests, please see this information on [Code Flow PRs](src/arcade/Documentation/UnifiedBuild/Codeflow-PRs.md).

Indivdual -> Individual 

I tend to avoid using tense like "now" as it quickly becomes dates.